dev/src/Command/NewComponentCommand.php

```diff
@@ -98,6 +106,16 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $new = NewComponent::fromProto($this->loadProtoContent($proto), $protoFile);
         $new->componentPath = $this->rootPath;
 
+        $unsafeTimeout = $input->getOption('timeout');
+
+        if (!is_numeric($unsafeTimeout)) {
+            throw new RuntimeException(
+                'Error: The timeout option must be a positive integer'
+            );
+        }
+
+        $this->timeout = (int) $unsafeTimeout;
```

Comment:
This is ok, but I wanted to mention since this a dev tool I'd just simplify this to one line


```suggestion
        $this->timeout = (int) $input->getOption('timeout');
```

Reply:
Aww but that's why I did it this way! I don't want a new dev trying this tools and seeing PHP complaining for a non number value and having to open the code to debug haha.